src/application/tester/test_runner.go

```diff
@@ -52,25 +57,12 @@ func (s *TestRunner) Shutdown() error {
 
 func (s *TestRunner) getRunOptions() (composerunner.RunOptions, error) {
 	dockerComposeProjectName := composebuilder.GetTestDockerComposeProjectName(s.AppContext.Config.Name)
-	dockerComposeConfigs, err := s.getDockerComposeConfigs()
-	if err != nil {
-		return composerunner.RunOptions{}, err
-	}
 	return composerunner.RunOptions{
 		AppDir:                   s.AppContext.Location,
-		DockerConfigs:            dockerComposeConfigs,
 		DockerComposeDir:         path.Join(s.AppContext.Location, "docker-compose"),
 		DockerComposeFileName:    s.BuildMode.GetDockerComposeFileName(),
 		DockerComposeProjectName: dockerComposeProjectName,
 		Writer:      s.Writer,
 		AbortOnExit: true,
 	}, nil
 }
-
-func (s *TestRunner) getDockerComposeConfigs() (types.DockerConfigs, error) {
-	return composebuilder.GetApplicationDockerConfigs(composebuilder.ApplicationOptions{
-		AppConfig: s.AppContext.Config,
-		AppDir:    s.AppContext.Location,
-		BuildMode: s.BuildMode,
```

Comment:
This build mode appears to have been lost

Reply:
Build mode is lost, but that's because `GenerateComposeFiles` is building for all build modes (and all docker-compose.yml files)